c/common/platform.h

```diff
@@ -180,6 +180,12 @@ To apply compiler hint, enclose the branching condition into macros, like this:
 #define BROTLI_UNUSED_FUNCTION static BROTLI_INLINE
 #endif
 
+#if BROTLI_GNUC_HAS_ATTRIBUTE(aligned, 2, 7, 0)
```

Comment:
This piece have been already added in your previous PR. So it is time to rebase =)

Reply:
Done.